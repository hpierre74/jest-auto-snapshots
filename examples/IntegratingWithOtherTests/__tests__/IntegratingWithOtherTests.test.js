import React from 'react';
import snap from 'jest-auto-snapshots';
import { shallow } from 'enzyme';
import IntegratingWithOtherTests from '../IntegratingWithOtherTests';

describe('IntegratingWithOtherTests', () => {
  snap(IntegratingWithOtherTests);

  describe('When button is clicked', () => {
    it('Should add a click', () => {
      const wrapper = shallow(<IntegratingWithOtherTests label="label" />);
      wrapper.find('button').simulate('click');
      expect(wrapper).toMatchSnapshot();
    });
  });

  /* ... any other functional/state tests you need to cover */
});
