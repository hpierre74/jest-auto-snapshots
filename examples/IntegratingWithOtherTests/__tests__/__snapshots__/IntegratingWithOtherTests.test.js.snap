// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IntegratingWithOtherTests jest-auto-snapshots > IntegratingWithOtherTests Matches snapshot when passed only required props 1`] = `
<div>
  <button
    onClick={[Function]}
    type="button"
  >
    Click Me
  </button>
  jest-auto-snapshots String Fixture
  :
  <span>
    0
  </span>
</div>
`;
